fuzz/href.cc

```diff
@@ -0,0 +1,33 @@
+#include <fuzzer/FuzzedDataProvider.h>
+
+#include <memory>
+#include <string>
+
+#include "ada.cpp"
```

Comment:
Prior to this these includes, we could add....

```
#define ADA_DEVELOPMENT_CHECKS 1
```

This would add additional checks.